### PR TITLE
Add AMP iFrame script to VJ Amp Include

### PR DESCRIPTION
**Pull request by chris-hinds**
No issue

**Overall change:**
Adds a missing script required for amp-iframes to load, allowing VJ includes to load on AMP Story pages _without a media player_. Story pages without a media player were unaffected as the script in included with the media player.

**Code changes:**
- Add amp-iframe extension script to `VjAmp` component using helmet, the script will be de-duped where a media player is present.
- Add test to check the script is added to the page head when the `VjAmp` component is used.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
Example of an affected page: http://localhost:7080/gahuza/23299713.amp
Example of a page with a media player that should still work: http://localhost:7080/mundo/23263889.amp ("Test Includes: Visual Journalism with iFrame" appears to not have an amp version when it should have "Test Includes: Visual Journalism with interactive content" does have an amp version and loads as expected)

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Include/amp/VjAmp.jsx [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
 import { shape, string } from 'prop-types';
 import { GridItemConstrainedMedium } from '#lib/styledGrid';
 
+const AmpHead = () => (
+  <Helmet>
+    <script
+      async
+      custom-element="amp-iframe"
+      src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"
+    />
```
Comment (rhenshaw56) [inline src/app/containers/Include/amp/VjAmp.jsx:13]: I was of the thought we usually added these [here](https://github.com/bbc/simorgh/blob/latest/src/server/Document/component.jsx#L68-L71), will it make any difference wherever we add these ?

Comment (chris-hinds) [reply src/app/containers/Include/amp/VjAmp.jsx:13]: Those are generic AMP scripts, those will be included on every AMP page. Things like amp-iframe are specific to the components being rendered so we don't want to add it on every page.

Comment (rhenshaw56) [reply src/app/containers/Include/amp/VjAmp.jsx:13]: Thanks Chris! That makes sense

**Review by rhenshaw56 — APPROVED** [review]

**Review by tochwill — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Pull request merged**